assignment-3/src/presentation/Main.java

```diff
@@ -0,0 +1,132 @@
+                    if (studentToUpdate != null) {
+                        System.out.print("Enter the new first name:");
+                        studentToUpdate.setFirstName(scanner.nextLine());
+                        System.out.print("Enter the new last name:");
+                        studentToUpdate.setLastName(scanner.nextLine());
+                        studentDao.update(studentToUpdate);
+                        System.out.println("___________________________");
+                    } else {
+                        System.out.println("Student not found with ID " + idToUpdate);
+                        System.out.println("___________________________");
+                    }
+                }
+                case LIST_MENU_ITEM -> {
+                    System.out.println("___________________________");
+                    System.out.printf("%-5s%-15s%-15s%n", "id", "firstName", "lastName");
+                    System.out.println("___________________________");
+                    for (Student student : studentDao.getAllStudents()) {
+                        System.out.println(student);
+                        System.out.println();
+                    }
+                }
+                case DELETE_MENU_ITEM -> {
+                    System.out.print("Enter the student ID to delete:");
+                    int idToDelete = scanner.nextInt();
+                    scanner.nextLine();
+                    studentDao.delete(idToDelete);
+                    System.out.println();
+                    System.out.println("___________________________");
+                }
+                case SORT_AND_SHOW -> {
+                    System.out.println("How would you like to sort the students?");
+                    System.out.println("1. By ID");
+                    System.out.println("2. By first name");
+                    System.out.println("3. By last name");
+                    System.out.print("Enter your choice:");
+                    int sortChoice = scanner.nextInt();
+                    List<Student> sortedStudents;
+                    switch (sortChoice) {
+                        case SORT_BY_ID -> sortedStudents = studentDao.getAllSortedById();
+                        case SORT_BY_FIRST_NAME -> sortedStudents = studentDao.getAllSortedByFirstName();
+                        case SORT_BY_LAST_NAME -> sortedStudents = studentDao.getAllSortedByLastName();
+                        default -> {
+                            System.out.println("Invalid sort choice.");
+                            sortedStudents = new ArrayList<>();
+                        }
+                    }
+                    System.out.println("___________________________");
+                    System.out.printf("%-5s%-15s%-15s%n", "id", "firstName", "lastName");
+                    System.out.println("___________________________");
+                    for (Student student : sortedStudents) {
```

Comment:
After you applied the correct property to sort (id, first name, or last name), you can also use `sortedStudents.forEach(student -> ...)` to print the result(s).